package protoFile.fight.LeaveCorps;
message LeaveCorpsRequest{
     required int32 id=1;//½ÇÉ«id
} 
message LeaveCorpsResponse{
     required bool result = 1;
     optional string message = 2;
}